### PR TITLE
25 create main application structure

### PR DESCRIPTION
**Pull request by Bacilika**

### CODE REVIEW

File: src/MAS.py
Comment (edvgi966) [inline src/MAS.py:1]: Missing Docstrings

**Review by edvgi966 — CHANGES_REQUESTED** [review]
MAS.py is missing docstrings

**Review by jakob123100 — CHANGES_REQUESTED** [review]
Add idea files to .gitignore

example add to .gitignore file:
.idea/

**Review by edvgi966 — APPROVED** [review]
Review pass.

**Review by jakob123100 — APPROVED** [review]
Looks good!

**Pull request merged**